go/vt/vtgate/planbuilder/builder.go

```diff
@@ -373,3 +373,26 @@ func buildShowTableStatusPlan(stmt *sqlparser.ShowTableStatus, vschema ContextVS
 		SingleShardOnly:   true,
 	}, nil
 }
+
+func buildLoadPlan(query string, vschema ContextVSchema) (engine.Primitive, error) {
+	keyspace, err := vschema.DefaultKeyspace()
+	if err != nil {
+		return nil, err
+	}
+
+	destination := vschema.Destination()
+	if destination == nil {
+		if keyspace.Sharded {
+			return nil, vterrors.Errorf(vtrpcpb.Code_INVALID_ARGUMENT, "set bypass destination first for a sharded keyspace")
```

Comment:
I'm not sure I'd know what to do if I got an error message called `"set bypass destination first for a sharded keyspace"`. What's a bypass destination? How do I set it?

Reply:
bypass queries are the ones where the user first issues "use ks@replica:-80" i.e. they give the exact shard that they want to run the query on. For now, we are only supporting Select into and Load for bypass queries and unsharded keyspaces because otherwise, the functionality of writing to and reading from files would have to reside in vtgate.

Reply:
Thanks for the explanation. I'd like to consider what the casual user would understand. Is it possible to elaborate a bit in the error message, to explain what needs to be done, or perhaps point to a relevant documentation, or give an example?

When I `grep` for `bypass` in the Vitess documentation, nothing comes up. Can we improve on the user experience?

Reply:
I'd also suggest we should come up with documentation on https://github.com/vitessio/website ; sorry, I don't mean to overload this PR, but I think it's a good opportunity to step up our documentation efforts.

Reply:
Yes absolutely, we should be adding documentation for this and maybe I'll do that for bypass queries so that when the user sees "set bypass destination firs"t, they can refer to bypass queries for better understanding.

Reply:
Alright, I'll change that

Reply:
I'll change the error message but would still like to add the documentation for bypass queries.